static/app/components/sidebar/sidebarItem.tsx

```diff
@@ -83,6 +84,15 @@ export type SidebarItemProps = {
    * Additional badge letting users know a tab is in beta.
    */
   isBeta?: boolean;
+  /**
+   * Is the sidebar collapsed or in mobile view
+   */
+  isFloatingSidebar?: boolean;
```

Comment:
Minor: After thinking for a bit, I think this would be less confusing if the component manually checked the orientation and collapse status here rather than passing this prop down. Generally, passing down compound conditionals like this is tough because it's a lot of context to keep in one's head

Feel free to ignore if this doesn't ring true

Reply:
Yeah, I agree, i think that sounds cleaner, i can also create a single hook that returns if the accordion should float